src/Propel/Generator/Behavior/Sortable/SortableBehaviorObjectBuilderModifier.php

```diff
@@ -289,6 +289,7 @@ public function objectFilter(string &$script, AbstractOMBuilder $builder): void
                     $script = str_replace($search, $replace, $script);
                 }
             } else {
+                /** @var string $scope */
                 $scope = current($this->behavior->getScopes());
```

Comment:
Just out of curiosity, why `/** @var string $scope */` and not `$scope = (string) current(...);`?

Reply:
In this context it looked like an impossible way to get another value here, so I just annotated it for humans and tools to know what to expect.
Sometimes casts are also not "valid" from Stan perspective since they might not work on certain return types.